target_chains/solana/sdk/js/pyth_solana_receiver/package.json

```diff
@@ -45,7 +45,7 @@
   "dependencies": {
     "@coral-xyz/anchor": "^0.29.0",
     "@noble/hashes": "^1.4.0",
-    "@pythnetwork/price-service-sdk": ">=1.6.0",
+    "@pythnetwork/price-service-sdk": "workspace:*",
```

Comment:
Not sure why `@pythnetwork/price-service-sdk` already set to `"workspace:*"`, but we need its `parseTwapMessage` from v1.8.0.